zanlib/classes/zandronum/rcon/ZandronumRcon.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Zanlib
+{
+    public partial class ZandronumRcon
```

Comment:
I think this is probably poorly named since it isn't the rcon protocol, it's more like server stats. The rcon stuff seems to be completely separate (but basically similar with the way the messages are sent/received etc).